src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/legacy/controller/dto/LegacyCreateSentence.kt

```diff
@@ -1,11 +1,13 @@
 package uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.legacy.controller.dto
 
+import io.swagger.v3.oas.annotations.media.Schema
 import java.time.LocalDate
 import java.util.UUID
 
 data class LegacyCreateSentence(
   val chargeUuids: List<UUID>,
-  val chargeNumber: String?,
+  @Schema(deprecated = true, description = "Deprecated. Will be removed soon.")
```

Comment:
Have marked this field as deprecated for now.. will remove it entirely once syson confirm they have stopped using it (they will put it in LegacyData)